### PR TITLE
Added API url in niginx

### PR DESCRIPTION
**Pull request by hpatel1994**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by poggdogg — APPROVED** [review]
Speeds up web page, adds API

File: src/frontend/citizen-portal/nginx.conf [hunk 1]
```diff
@@ -14,3 +22,7 @@ error_page   500 502 503 504  /50x.html;
 location = /50x.html {
   root   /usr/share/nginx/html;
 }
+
+location /api/v1/ {
```
Comment (pbolduc) [inline src/frontend/citizen-portal/nginx.conf:26]: Did you verify this is correct?  I didn't expect `/v1/`

Comment (poggdogg) [reply src/frontend/citizen-portal/nginx.conf:26]: we could remove it, no big deal.
Make it just /api

**Comment by poggdogg** [comment]
HTTP still cuz we have no vanity cert for HTTPS to work.

**Pull request merged**